apps/cowswap-frontend/package.json

```diff
@@ -11,7 +11,7 @@
     "patch-package": "npx patch-package",
     "cosmos:clear": "rm -rf ./public/cosmos",
     "cosmos:run": "yarn run _cosmos:run",
-    "build:cosmos": "cross-env NODE_ENV=development npx cosmos-export",
+    "build:cosmos": "NODE_ENV=development npx cosmos-export",
```

Comment:
`cross-env` was never implicitly added as a dependency in the project and got lost with the upgrades. Since no one is using Windows in the team, I think it's not a big deal if we remove it

Reply:
We don't but external contributors might.
Even though I never cared for this, might be a good practice to be sure to support as many envs as possible.
After all, this is a dev dependency and shouldn't impact the production build.
I vote then for adding `cross-env` as devDependency if not much of a hassle.

Reply:
You convinced me, added it back